### PR TITLE
BZ2048180: add a note to capture a limitation with respect to image streams and the internal registry

### PR DESCRIPTION
**Pull request by chinmayi-chandrasekar**
For 4.6+
https://bugzilla.redhat.com/show_bug.cgi?id=2048180

[Doc preview](https://deploy-preview-41488--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/managing_images/using-image-pull-secrets.html)

Requires ack from @xiuwang

### CODE REVIEW

File: openshift_images/managing_images/using-image-pull-secrets.adoc [hunk 1]
```diff
@@ -8,6 +8,11 @@ toc::[]
 
 If you are using the {product-title} internal registry and are pulling from image streams located in the same project, then your pod service account should already have the correct permissions and no additional action should be required.
 
+[NOTE]
+====
+When you create a pod service account or a namespace, wait until it is provisioned with a docker pull secret; if you create a pod before its service account is fully provisioned, the pod fails to access the {product-title} internal registry.
```
Comment (xiuwang) [inline openshift_images/managing_images/using-image-pull-secrets.adoc:13]: wait until the service account is provisioned with a docker pull secret

Comment (chinmayi-chandrasekar) [reply openshift_images/managing_images/using-image-pull-secrets.adoc]: @xiuwang Comment addressed.

**Comment by xiuwang** [comment]
LGTM

**Comment by ctauchen** [comment]
This note seems out of place in its current location. Is there a better place to put this information?

The note breaks up paragraphs that go together. Note the "However, ..." in the paragraph after the note.

It seems to me that this information would be best placed when we tell people how to add a pull secret to the a pod service account. 

Perhaps somewhere in here?
https://deploy-preview-41488--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/managing_images/using-image-pull-secrets.html#images-allow-pods-to-reference-images-from-secure-registries_using-image-pull-secrets

Might be worth talking to @bmcelvee to find the best spot for this.

**Comment by bmcelvee** [comment]
> https://deploy-preview-41488--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/managing_images/using-image-pull-secrets.html#images-allow-pods-to-reference-images-from-secure-registries_using-image-pull-secrets

Makes sense to me, thanks! :)

**Comment by chinmayi-chandrasekar** [comment]
> This note seems out of place in its current location. Is there a better place to put this information?
> 
> The note breaks up paragraphs that go together. Note the "However, ..." in the paragraph after the note.
> 
> It seems to me that this information would be best placed when we tell people how to add a pull secret to the a pod service account.
> 
> Perhaps somewhere in here? https://deploy-preview-41488--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/managing_images/using-image-pull-secrets.html#images-allow-pods-to-reference-images-from-secure-registries_using-image-pull-secrets
> 
> Might be worth talking to @bmcelvee to find the best spot for this.

@ctauchen @bmcelvee Isn't this - https://deploy-preview-41488--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/managing_images/using-image-pull-secrets.html#images-allow-pods-to-reference-images-from-secure-registries_using-image-pull-secrets talking about registries other than the internal registry? That note is specific to the internal registry.

**Comment by ctauchen** [comment]
> > This note seems out of place in its current location. Is there a better place to put this information?
> > The note breaks up paragraphs that go together. Note the "However, ..." in the paragraph after the note.
> > It seems to me that this information would be best placed when we tell people how to add a pull secret to the a pod service account.
> > Perhaps somewhere in here? https://deploy-preview-41488--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/managing_images/using-image-pull-secrets.html#images-allow-pods-to-reference-images-from-secure-registries_using-image-pull-secrets
> > Might be worth talking to @bmcelvee to find the best spot for this.
> 
> @ctauchen @bmcelvee Isn't this - https://deploy-preview-41488--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/managing_images/using-image-pull-secrets.html#images-allow-pods-to-reference-images-from-secure-registries_using-image-pull-secrets talking about registries other than the internal registry? That note is specific to the internal registry.

Okay, so maybe better here? https://deploy-preview-41488--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/managing_images/using-image-pull-secrets.html#images-allow-pods-to-reference-images-across-projects_using-image-pull-secrets

**Comment by xiuwang** [comment]
> Okay, so maybe better here? https://deploy-preview-41488--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/managing_images/using-image-pull-secrets.html#images-allow-pods-to-reference-images-across-projects_using-image-pull-secrets

It makes sense,looks better.

**Comment by chinmayi-chandrasekar** [comment]
> 

Comment addressed.

**Pull request merged**